packages/web-app/src/views/check-and-confirm.njk

```diff
@@ -0,0 +1,117 @@
+{% extends "layouts/form.njk" %}
+
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "components/address.njk" import address %}
+{% from "components/unorderedList.njk" import unorderedList %}
+{% from "components/button.njk" import button %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% set yourDetails = appealData.aboutYouSection.yourDetails %}
+{% set originalApplicationFile = appealData.requiredDocumentsSection.originalApplication.uploadedFile %}
+{% set decisionLetterFile = appealData.requiredDocumentsSection.decisionLetter.uploadedFile %}
+{% set appealStatementFile = appealData.yourAppealSection.appealStatement.uploadedFile %}
+{% set outcomeDetails = reviewOutcome.outcomeDetails %}
+{% set invalidAppealDetails = reviewOutcome.invalidAppealDetails %}
+{% set validAppealDetails = reviewOutcome.validAppealDetails %}
+{% set completedError = errors['check-and-confirm-completed'].msg %}
+
+{% macro renderReason(reason) %}
+  <p class="govuk-body govuk-!-margin-bottom-2">{{ reason }}</p>
```

Comment:
I don't think we need to put this text in a `p` tag because it's part of the `dl` created by `govukSummaryList`. The other `dd` values by default aren't wrapped in a `p` tag so not using one here would keep these consistent and keep the margin bottom the same for each, which I see that you've tried to correct with `govuk-!-margin-bottom-2` but it is still slightly larger than the other values.